lib/shoulda/matchers/active_record/define_enum_for_matcher.rb

```diff
@@ -111,6 +113,18 @@ def enum_values_match?
           expected_enum_values.empty? || actual_enum_values == expected_enum_values
         end
 
+        def matched_column
+          @_matched_column ||= begin
```

Comment:
ActiveRecord provides a `columns_hash` method you can use if you want the Column object for a particular attribute. So you can replace this with `@model.class.columns_hash[attribute_name.to_s]` instead, here.